.gitignore

```diff
@@ -7,6 +7,6 @@ __pycache__/
 .pytest_cache
 *.DS_Store
 check_data/
-models/
+/models/
```

Comment:
I don't think it works like that
try just `models`?

Reply:
Well there are more than one folders named 'models' so I just want only the one that's in the root directory and keeps the trained models. My change seems to work.